src/UserActivity.js

```diff
@@ -23,105 +23,160 @@ import Timer from './utils/Timer';
 // such as READ_MARKER_INVIEW_THRESHOLD_MS,
 // READ_MARKER_OUTOFVIEW_THRESHOLD_MS,
 // READ_RECEIPT_INTERVAL_MS in TimelinePanel
-const CURRENTLY_ACTIVE_THRESHOLD_MS = 2 * 60 * 1000;
+const CURRENTLY_ACTIVE_THRESHOLD_MS = 700;
+const CURRENTLY_PASSIVE_THRESHOLD_MS = 2 * 60 * 1000;
 
 /**
  * This class watches for user activity (moving the mouse or pressing a key)
  * and starts/stops attached timers while the user is active.
+ *
+ * There are two classes of 'active' a user can be: 'active' and 'passive':
```

Comment:
Mmm, 'focused' makes me think it's more just about tracking whether the window has focus. We could go with Bruno's 'activeActive' and 'passiveActive', or 'activeNow' and 'activeRecently'?

Reply:
Hmm, `activeNow` vs. `activeRecently` sounds like the clearest so far to me.

Reply:
Was just about to change this and then realised this would make functions called `userCurrentlyActiveNow()` and, even better, `userCurrentlyActiveRecently()`. I'll have a think...

Reply:
Maybe `userActiveNowish()` and `userActiveRecently()`?

Reply:
Yeah, I was thinking maybe just `userActiveNow()`

Reply:
Right, that could work as well.